easybib/libraries/sns.rb

```diff
@@ -0,0 +1,31 @@
+require 'json'
+
+module EasyBib
+  module SNS
+    def self.sns_notify(node, body)
+      require 'aws-sdk'
+
+      if node.nil?
+        Chef::Log.error 'Missing argument: node (AWS instance name)!'
```

Comment:
@baccenfutter This still will continue and therefore lead to a broken deploy. better return here, too :)